### PR TITLE
Jason-Belgard

### PR DESCRIPTION
**Pull request by jbelgard**

### CODE REVIEW

File: src/reducers/index.js [hunk 1]
```diff
@@ -12,9 +12,9 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+      return {...state, count: initialState.count++}
```
Comment (sebaghdev) [inline src/reducers/index.js:15]: Go to next line.

```
return {
    ...state,
    count: initialState.count++
}
```

File: src/reducers/index.js [hunk 2]
```diff
@@ -12,9 +12,9 @@ const initialState = {
...
     case DECREMENT:
-    // Fill in the body of this case
+      return {...state, count: initialState.count--}
```
Comment (sebaghdev) [inline src/reducers/index.js:17]: Same here.

```
return {
    ...state,
    count: initialState.count--
}
```

File: src/reducers/index.js (hunk 1 above)
Comment (jbelgard) [reply src/reducers/index.js:15]: Ok, fixed

File: src/reducers/index.js (hunk 2 above)
Comment (jbelgard) [reply src/reducers/index.js:17]: Ok, fixed

**Pull request merged**